learning/scripting/visual_script/visual_script_basics.rst

```diff
@@ -363,15 +388,16 @@ Most of the time they are integer or float.
 
 .. image:: img/visual_script36.png
 
-Of interest here are mainly three nodes. The first one is "Constant" which allows to select any value of any type as constant,
-from an integer (42) to a String ("Hello!"). In general this node is not used that often because of default input
+
+Of interest there are mainly three nodes. The first one is "Constant" which allows you to select any value of any type as constant, from an integer (42) to a String ("Hello!"). In general this node is not used that often because of default input
```

Comment:
Remove the "Of interest..." sentence.

If it's not used that often, why are we discussing it?

Reply:
Here I'm confused, as Juan has "but it's good to know it exists.". To me, that would mean it should be part of the documentation, even if it's not used often. Or, what would you recommend doing here?